### PR TITLE
feat: add poe_filter

### PR DESCRIPTION
**Pull request by ObserverOfTime**
This adds support for [PoE item filters](https://www.pathofexile.com/item-filter/about).

If you think that's too niche, you can close the PR.

### CODE REVIEW

**Comment by clason** [comment]
> If you think that's too niche, you can close the PR.

_Nothing_ is too niche for nvim-treesitter (we have cooklang and beancount...)

File: lua/nvim-treesitter/parsers.lua [hunk 1]
```diff
@@ -1443,6 +1443,17 @@ list.ini = {
...
 
+list.poe_filter = {
+  install_info = {
+    url = "https://github.com/ObserverOfTime/tree-sitter-poe-filter",
+    files = { "src/parser.c" },
+  },
+  maintainers = { "@ObserverOfTime" },
+  experimental = true,
+  filetype = "poefilter",
+  readme_name = "PoE item filter",
```
Comment (theHamsta) [inline lua/nvim-treesitter/parsers.lua:1454]: maybe also add the link you gave us :smile:

**Review by theHamsta — APPROVED** [review]

File: lua/nvim-treesitter/parsers.lua (hunk 1 above)
Comment (ObserverOfTime) [reply lua/nvim-treesitter/parsers.lua:1454]: As a comment?

Comment (theHamsta) [reply lua/nvim-treesitter/parsers.lua:1454]: so that it is visible in the README. So maybe a markdown link or just the link in parens.

Comment (clason) [reply lua/nvim-treesitter/parsers.lua:1454]: I think it's enough to spell it out ("Path of Exile").

**Review by theHamsta — APPROVED** [review]

**Pull request merged**